src/Meshes/thinInstanceMesh.ts

```diff
@@ -74,6 +74,15 @@ declare module "./mesh" {
          */
         thinInstanceBufferUpdated(kind: string): void;
 
+        /**
+         * Applies a partial update to a buffer directly on the GPU
+         * Note that the buffer located on the CPU is NOT updated! It's up to you to update it (or not) with the same data you pass to this method
+         * @param kind name of the attribute to update. Use "matrix" to update the buffer of matrices
+         * @param data the data to set in the GPU buffer
+         * @param offset the offset in the GPU buffer where to update the data
+         */
+        thinInstancePartialBufferUpdate(kind: string, data: Float32Array, offset: number): void;

```

Comment:
We should later add some usage in docs, other than that great feature, was waiting for this, thank you 👍

Reply:
You're try, it's done here: https://github.com/BabylonJS/Documentation/pull/2026